src/client.ts

```diff
@@ -185,6 +190,18 @@ export class Client {
     return this._environmentData;
   }
 
+  /**
+   * Gets the host url prefix for current app
+   */
+  public asHostUrl(clientRoute: string) {
```

Comment:
minor nit: we have so far kept return types on all methods.  So :void in this case.

Reply:
Also can we add some tests to this

Reply:
Will add test cases